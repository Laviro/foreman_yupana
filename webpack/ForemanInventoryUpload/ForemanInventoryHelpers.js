import { foremanUrl } from '../ForemanRhCloudHelpers';

export const inventoryUrl = path =>
  foremanUrl(`/foreman_inventory_upload/${path}`);

export const getInventoryDocsUrl = () =>
  'https://access.redhat.com/products/subscription-central';
