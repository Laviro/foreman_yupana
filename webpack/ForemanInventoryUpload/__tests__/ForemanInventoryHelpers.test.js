import { testSelectorsSnapshotWithFixtures } from '@theforeman/test';
import { inventoryUrl } from '../ForemanInventoryHelpers';

const fixtures = {
  'should return inventory Url': () => inventoryUrl('test_path'),
};

describe('ForemanInventoryUpload helpers', () =>
  testSelectorsSnapshotWithFixtures(fixtures));
