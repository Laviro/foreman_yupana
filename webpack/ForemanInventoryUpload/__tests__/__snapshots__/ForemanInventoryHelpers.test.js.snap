// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForemanInventoryUpload helpers should return inventory Url 1`] = `"/foreman_inventory_upload/test_path"`;

exports[`ForemanInventoryUpload helpers should return inventory docs url 1`] = `"https://access.redhat.com/products/subscription-central"`;
